### ISSUE TITLE
Resolve deprecation warnings

### ISSUE THREAD

**Issue by mar235av** [UWIT-IAM/uw-idp-custom#49]
The most significant part of the upgrade to Shibboleth v5.X is that some components have been deprecated. Fortunately, the last 4.X version set warning messages for all such cases. We need to identify our use of deprecated features and modify our deployment accordingly.

- [x] Scan production log files for deprecation warnings.
- [x] Restart a non-prod instance and note any deprecation warnings on start-up.
- [ ] Resolve each warning by making the appropriate changes.
- [ ] Deploy to production (still at v4.X) and monitor logs to verify all issues are resolved.

**Comment by mar235av** [comment]
Scanned and found five deprecation warnings:

2024-06-18 00:00:01,338 - WARN [DEPRECATED:135] - Spring bean 'shibboleth.HttpServletRequest': This will be removed in the next major version of this software; replacement is shibboleth.HttpServletRequestSupplier
2024-06-18 08:53:56,865 - WARN [DEPRECATED:135] - Spring bean 'shibboleth.HttpServletResponse': This will be removed in the next major version of this software; replacement is shibboleth.HttpServletRespoonseSupplier
2024-06-18 11:21:52,275 - WARN [DEPRECATED:128] - xsi:type 'SAML2NameID', (file [/data/local/idp/conf/attribute-resolver.xml]): This will be removed in the next major version of this software; replacement is (none)
2024-06-18 11:21:56,045 - WARN [DEPRECATED:135] - Spring bean 'shibboleth.NonCachingHttpClient': This will be removed in the next major version of this software; replacement is shibboleth.HttpClientFactory
2024-06-18 11:21:57,653 - WARN [DEPRECATED:110] - property 'idp.service.attribute.registry.namingRegistry' is no longer supported

**Comment by mar235av** [comment]
Fixed with several steps:
1. The first two warning were fixed by changing definitions in conf/global.xml, requiring compatible changes to conf/authn/mfa-authn-config.xml, per release notes for version 4.0.0 and HttpClient configuration.
2. The third warning was actually not deprecated in version 5.X, contrary to prior notice. We should investigate this, but this doe snot impact the upgrade.
3. The fourth warning was fixed by changing the reference in conf/global.xml as noted in the warning.
4. The last warning was a result of an old properties file inadvertently imported into the configuration. I removed that which resolved this warning.
